benchmark/tpcds.md

```diff
@@ -92,110 +92,111 @@ TPC-DS 99 test query statements : [TPC-DS-Query-SQL](https://github.com/apache/d
-| query65   | 5410                          | 5971                     |
-| query66   | 484                           | 603                      |
-| query67   | 26347                         | 34052                    |
-| query68   | 1422                          | 1428                     |
-| query69   | 654                           | 808                      |
-| query70   | 2285                          | 4462                     |
-| query71   | 650                           | 1006                     |
-| query72   | 4324                          | 4717                     |
-| query73   | 500                           | 558                      |
-| query74   | 6678                          | 14127                    |
-| query75   | 3734                          | 6312                     |
-| query76   | 1835                          | 1870                     |
-| query77   | 382                           | 496                      |
-| query78   | 19923                         | 23091                    |
-| query79   | 3061                          | 4090                     |
-| query80   | 851                           | 1559                     |
-| query81   | 565                           | 960                      |
-| query82   | 242                           | 221                      |
-| query83   | 254                           | 415                      |
-| query84   | 203                           | 131                      |
-| query85   | 364                           | 444                      |
-| query86   | 651                           | 931                      |
-| query87   | 8972                          | 8554                     |
-| query88   | 4095                          | 5202                     |
-| query89   | 508                           | 480                      |
-| query90   | 233                           | 322                      |
-| query91   | 174                           | 159                      |
-| query92   | 62                            | 59                       |
-| query93   | 1601                          | 1618                     |
-| query94   | 297                           | 297                      |
-| query95   | 1240                          | 27354                    |
-| query96   | 508                           | 847                      |
-| query97   | 5449                          | 11528                    |
-| query98   | 382                           | 287                      |
-| query99   | 1410                          | 2147                     |
-| **Total** | **264028**                    | **487990**               |
+Here we use Apache Doris 2.1.7-rc03 and Apache Doris 2.0.16-rc01 for comparative testing. In the test, we use Query Time(ms) as the main performance indicator.
+The test results are as follows: (Apache Doris 2.0.16-rc01 q78 q79 failed to execute due to memory issues and was removed when calculating the total sum)
```

Comment:
"memory issue" better changed to other words, e.g, "lack of latest memory optimization", etc.